PacketDelivery-VC-ReferenceExample/data-consumer/README.md

```diff
@@ -0,0 +1,173 @@
+
+|   |   |
+|---|---|
+|DID of consumer's authentication authority | did:key:z6MkigCEnopwujz8Ten2dzq91nvMjqbKQYcifuZhqBsEkH7g |
+|DID of provider's authentication authority | did:key:z6MkigCEnopwujz8Ten2dzq91nvMjqbKQYcifuZhqBsEkH7h |
+|Password for 'normal-user'   | normal-user  |
+|Password for 'premium-user'   | premium-user  |
+
+## Validation of components
+### WaltId
+
+WaltId is the component that is used for initializing verifiable credentials that match a given template using a defined key. The template is provided on startup and the key is registered by the Keycloak VC Issuer Plugin.
+
+To validate that the template was successfully imported, following command can be used in the k8s cluster:
+
+```
+curl -X 'GET' 'http://waltid-vcwaltid:7001/v1/templates/PacketDeliveryService' -H 'accept: application/json'
+```
+with an expected output of
+```
+{
+    "type": [
+        "VerifiableCredential",
+        "PacketDeliveryService"
+    ],
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1"
+    ],
+    "id": "urn:uuid:3add94f4-28ec-42a1-8704-4e4aa51006b4",
+    "issuer": "did:ebsi:2A9BZ9SUe6BatacSpvs1V5CdjHvLpQ7bEsi2Jb6LdHKnQxaN",
+    "issuanceDate": "2021-08-31T00:00:00Z",
+    "issued": "2021-08-31T00:00:00Z",
+    "validFrom": "2021-08-31T00:00:00Z",
+    "credentialSchema": {
+        "id": "https://raw.githubusercontent.com/hesusruiz/dsbamvf/main/schemas/PacketDeliveryService/2022-10/schema.json",
+        "type": "FullJsonSchemaValidator2021"
+    },
+    "credentialSubject": {
+        "id": "did:ebsi:2AEMAqXWKYMu1JHPAgGcga4dxu7ThgfgN95VyJBJGZbSJUtp",
+        "familyName": "User",
+        "firstName": "Some",
+        "roles": [
+            {
+                "names": [
+                    "someRole"
+                ],
+                "target": "did:some:id"
+            }
+        ]
+    }
+}
+```
+
+To validate that the signing key was successfully registered by the Keycloak VC Issuer Plugin, the following command can be used
+
+```
+curl -X 'GET' 'http://waltid-vcwaltid:7002/keys/z6MkigCEnopwujz8Ten2dzq91nvMjqbKQYcifuZhqBsEkH7g'   -H 'accept: application/json'
+```
+with an expected output of
+```
+{
+    "algorithm": "EdDSA_Ed25519",
+    "cryptoProvider": "SUN",
+    "keyId": {
+        "id": "z6MkigCEnopwujz8Ten2dzq91nvMjqbKQYcifuZhqBsEkH7g"
+    },
+    "keyPair": {},
+    "keysetHandle": null
+}
+```
+
+### Keycloak
+
+Keycloak IDM is used for managing the users in the consumer space and offering verifiable credentials for accessing the data producer.
+
+A GUI for providing Verifiable Credentials to be imported into the wallet can be found under
+
+https://kc-consumer.domain.org/realms/fiware-server/account/#/verifiable-credentials
+
+and are provided via QR code like
+![CredentialsQR](./img/credential_issuance.png "CredentialsQR")
+
+Alternatively its possible to request the signed Verifiable Credentials via the REST API:
```

Comment:
I would rather not mention this, it will be removed in the near future. It unsecure, proprietary and was only there due to the shortcommings of our demo wallet.